staging/src/k8s.io/component-base/metrics/prometheus/feature/metrics.go

```diff
@@ -34,9 +38,6 @@ var (
 		},
 		[]string{"name", "stage"},
 	)
-)
-
-func init() {
 	legacyregistry.MustRegister(featureInfo)
 }
 
```

Comment:
do we need to protect RecordFeatureInfo from a nil?

Reply:
Done